group-actors/src/main/java/org/sunbird/service/MemberServiceImpl.java

```diff
@@ -85,7 +84,7 @@ public void handleMemberOperations(Map memberOperationMap, String groupId, Strin
   public Response handleMemberAddition(
       List<Map<String, Object>> memberList, String groupId, String contextUserId)
       throws BaseException {
-    logger.info("Number of members to be added are: {}", memberList.size());
+    logger.info("Number of members to be added to the group {} are {}", groupId, memberList.size());
```

Comment:
do we have similar message for modified, removed?

Reply:
No, I added now